### PR TITLE
Added registry access requirement

### PR DESCRIPTION
**Pull request by gaurav-nelson**
- Added registry access under requirements 
- Fixed Kubernetes spell

### CODE REVIEW

File: modules/acs-requirements.adoc [hunk 1]
```diff
@@ -36,3 +36,9 @@ However, you can use another storage type if you do not have SSDs available.
 Use the `helm version` command to verify the version of Helm you have installed.
 * The {ocp} CLI (`oc`).
 * You must have the required permissions to configure deployments in the Central cluster.
+* You must have access to the Red Hat Container Registry. See link:https://access.redhat.com/RegistryAuthentication[Red Hat Container Registry Authentication] for information about downloading images from `registry.redhat.io`.
++
+[NOTE]
+====
+If you are not a Red Hat customer and purchased the Stackrox Kubernetes Security Platform before the acquisition. You can use StackRox’s container registry at `stackrox.io`. Contact mailto:support@stackrox.com[support@stackrox.com] to enable access to the registry.
```
Comment (mikemckiernan) [inline modules/acs-requirements.adoc:43]: First sentence is a frag, right?

If you are not a Red Hat customer and purchased the Stackrox Kubernetes Security Platform before the acquisition, you can use the Stackrox container registry at `stackrox.io`.

**Review by mikemckiernan — COMMENTED** [review]
One comment.  If this needs a label like "Next release", please let me know.

File: modules/acs-requirements.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply modules/acs-requirements.adoc]: Ah 🦊 fixed it. Thanks.

**Comment by gaurav-nelson** [comment]
Thanks @mikemckiernan It doesn't need any labels. This is something I missed when porting the docs over from https://help.stackrox.com/docs/.

**Pull request merged**